### PR TITLE
delete intermediate chain cert for ht

### PR DESCRIPTION
**Pull request by umjnega**

### CODE REVIEW

**Comment by rrotter** [comment]
nebula::profile::ssl_keypair is only used on one host besides the hathitrust apache hosts, so removing it there only requires getting a new full chain cert.

**Comment by rrotter** [comment]
More on this in [delete-nebula-profile-sslkeypair-chain-crt](/mlibrary/nebula/tree/delete-nebula-profile-sslkeypair-chain-crt). Waiting until standup to push those changes to this branch.

**Review by aelkiss — APPROVED** [review]
Seems fine to me if serving the full chain from the .crt file works OK.

File: spec/fixtures/hiera/hathitrust.yaml [hunk 1]
```diff
@@ -59,7 +59,7 @@ hathitrust::networks::staff:
   - "%{alias('networks::one')}"
   - "%{alias('networks::two')}"
 
-nebula::profile::ssl_keypair::chain_crt: 'intermediate_ca.crt'
+nebula::profile::ssl_keypair::chain_crt: 'bad_cert.crt'
```
Comment (botimer) [inline spec/fixtures/hiera/hathitrust.yaml:62]: With the removal of the expectation, I think this is now extraneous. Not harmful, but unused.

**Review by botimer — APPROVED** [review]

File: spec/fixtures/hiera/hathitrust.yaml (hunk 1 above)
Comment (rrotter) [reply spec/fixtures/hiera/hathitrust.yaml:62]: yep, purged this in a later commit.

**Review by rrotter — APPROVED** [review]

**Pull request merged**